python/cudf/cudf/core/column/string.py

```diff
@@ -3462,10 +3465,13 @@ def findall(
         1  <NA>  <NA>
         2     b     b
         """
-        if flags != 0:
-            raise NotImplementedError("`flags` parameter is not yet supported")
+        if isinstance(pat, re.Pattern):
+            flags = pat.flags & ~re.U
+            pat = pat.pattern
```

Comment:
Was re.Pattern previously erroring silently? In the absence of flags we should already have been extracting the string from the Pattern right?